### PR TITLE
Used assign and unassign for metrics exporter

### PR DESCRIPTION
**Pull request by shinji62**

### CODE REVIEW

File: cmd/metrics_exporter/metrics_exporter.go [hunk 1]
```diff
@@ -175,9 +175,9 @@ var deleteMetricsExporterCmd = &cobra.Command{
 }
 
 var removeMetricsExporterFromClusterCmd = &cobra.Command{
-	Use:   "remove-from-cluster",
-	Short: "Remove Metrics Exporter Config from Cluster",
-	Long:  "Remove Metrics Exporter Config from Cluster",
+	Use:   "unassign",
```
Comment (posriniv) [inline cmd/metrics_exporter/metrics_exporter.go:178]: Isn't this a breaking change since we already released a version?

**Review by posriniv — CHANGES_REQUESTED** [review]

File: cmd/metrics_exporter/metrics_exporter.go (hunk 1 above)
Comment (posriniv) [reply cmd/metrics_exporter/metrics_exporter.go:178]: Well, I agree, I will let @shinji62 take the call. Just one concern, let' say someone installed v0.1.15 and used this command and later they upgraded to v0.1.6, something which worked earlier will stop working and that doesn't look good.

Comment (shinji62) [reply cmd/metrics_exporter/metrics_exporter.go]: Added alias that old command should work

**Review by posriniv — APPROVED** [review]
Looks good to me!

**Pull request merged**